Packs/CommonScripts/Scripts/IsIPPrivate/IsIPPrivate.py

```diff
@@ -0,0 +1,57 @@
+from ipaddress import IPv4Address, IPv4Network
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def check_ip_internal(ip, ranges):
+    for cidr in ranges:
+        if IPv4Address(ip) in IPv4Network(cidr):
+            return True
+    return False
```

Comment:
```suggestion
    return any((IPv4Address(ip) in IPv4Network(cidr) for cidr in ranges))
```

Reply:
For this one I've made a small change to accommodate for exact IPs (not CIDR), so if the IP is an exact match to an IP in the list it will also return True for that IP

Reply:
where is the change? the current code seems untouched

Reply:
For some reason it took time to change. See now

Reply:
still don't see a change. Did you push?

Reply:
I pushed the changes, maybe you're looking at an old commit? This is the commit where I fixed it to be one line: https://github.com/demisto/content/pull/24077/commits/2a787c8cf3c8866d47c780bee3f3c3237970ac9d